google/cloud/bigtable/instance.py

```diff
@@ -644,6 +646,182 @@ def list_tables(self):
 
         return result
 
+    def backup(self, backup_id, cluster_id=None, table=None, expire_time=None):
+        """ Factory to create a Backup within this Instance.
+
+        :type backup_id: str
+        :param backup_id: The ID of the Backup to be created.
+
+        :type cluster_id: str
+        :param cluster_id: (Optional) The ID of the Cluster. Required for
+                           calling 'delete', 'exists' etc. methods.
+
+        :type table: :class:`~google.cloud.bigtable.table.Table`
+        :param table: (Optional) The Table create the Backup from.
+            Required, if the `create` method needs to be called.
+
+        :type expire_time: :class:`datetime.datetime`
+        :param expire_time: (Optional) The expiration time of this new Backup.
+            Required, if the `create` method needs to be called.
+        """
+        try:
+            return backup.Backup(
+                backup_id,
+                self,
+                cluster_id=cluster_id,
+                table_id=table.name.split('/')[-1],
+                expire_time=expire_time
+            )
+        except AttributeError:
+            return backup.Backup(
+                backup_id, self,
+                cluster_id=cluster_id,
+                table_id=table,
+                expire_time=expire_time
+            )
+
+    def list_backups(self, cluster_id=None,
+                     filter_=None, order_by=None, page_size=None):
+        """ List Backups for the Instance.
+
+        :type cluster_id: str
+        :param cluster_id: (Optional) Specifies a single cluster to list
+                        Backups from. If none is specified, the returned list
```

Comment:
``` `None` ```

Reply:
In this case we're dealing with English, not Python. `'None' is specified` would mean that we actually specified it as `None`, whereas this way implies that no such parameter was provided.